### PR TITLE
Updated fragment import to remove depreciation warning

### PR DESCRIPTION
**Pull request by AhtishamShahid**
## [PROD-1388](https://openedx.atlassian.net/browse/PROD-1388)

Updated fragment import to remove deprecation warning to avoid unnecessary logs in Splunk

### CODE REVIEW

File: lti_consumer/lti_consumer.py [hunk 1]
```diff
@@ -65,7 +65,7 @@
 from webob import Response
 from xblock.core import List, Scope, String, XBlock
 from xblock.fields import Boolean, Float, Integer
-from xblock.fragment import Fragment
+from web_fragments import Fragment
```
Comment (awaisdar001) [inline lti_consumer/lti_consumer.py]: Is it same as `from web_fragments.fragment import Fragment`?

File: lti_consumer/lti_consumer.py [hunk 2]
```diff
@@ -927,7 +927,7 @@ def _get_context_for_template(self):
         # This lets all plaintext through.
         allowed_tags = bleach.sanitizer.ALLOWED_TAGS + ['img']
         allowed_attributes = dict(bleach.sanitizer.ALLOWED_ATTRIBUTES, **{'img': ['src', 'alt']})
-        sanitized_comment = bleach.clean(self.score_comment, tags=allowed_tags, attributes=allowed_attributes)
+        sanitized_comment = bleach.clean('<img src=http://lagunita.stanford.edu/c4x/Philosophy/LPL/asset/ex_correct_summtable.png> Congratulations, your solution is correct.', tags=allowed_tags, attributes=allowed_attributes)
```
Comment (awaisdar001) [inline lti_consumer/lti_consumer.py]: Why this has changed?

File: lti_consumer/templates/html/student.html [hunk 3]
```diff
@@ -78,12 +78,11 @@ <h3 class="error_message">
     </h3>
 % endif
 
-% if has_score and comment:
     <h4 class="problem-feedback-label">Feedback on your work from the grader:</h4>
     <div class="problem-feedback">
         ## sanitized with bleach in view
         ${comment}
     </div>
-% endif
```
Comment (awaisdar001) [inline lti_consumer/templates/html/student.html]: Can you add why this conditional block is removed.

Comment (AhtishamShahid) [reply lti_consumer/templates/html/student.html]: I was testing some stuff. Fixed now.

**Pull request merged**